### PR TITLE
[WIP] Update githubteacher's slide

### PR DESCRIPTION
**Pull request by githubteacher**
**TL;DR** Summarize the changes

## Ask leading questions
- Hey does this look like the right feature to be adding? 

## Create a task list 
- [ ] Tasks to be completed

cc: @beardofedu for your awareness

### CODE REVIEW

**Comment by githubteacher** [comment]
New comment

**Review by carpnick — APPROVED** [review]
👍

File: _posts/0000-01-21-githubteacher.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Making a miner change
```
Comment (carpnick) [inline _posts/0000-01-21-githubteacher.md:6]: @githubteacher  Looks great.

File: _posts/0000-01-21-githubteacher.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
 ![minertocat](https://octodex.github.com/images/minertocat.png)
```
Comment (carpnick) [inline _posts/0000-01-21-githubteacher.md:10]: Word

**Review by carpnick — COMMENTED** [review]
This needs more work - Fix it 🍰 🚯

File: _posts/0000-01-21-githubteacher.md (hunk 1 above)
Comment (githubteacher) [reply _posts/0000-01-21-githubteacher.md:6]: > This needs more work - Fix it 🍰 🚯

This is comedy gold, I don't know what you're talking about.

**Review by carpnick — APPROVED** [review]
👍 🎉 

Also ✋

**Pull request merged**